window/src/os/wayland/window.rs

```diff
@@ -1312,6 +1357,14 @@ impl CompositorHandler for WaylandState {
         _surface: &wayland_client::protocol::wl_surface::WlSurface,
         _new_factor: i32,
     ) {
+        // TODO: uncomment after upgrading SCTK to 0.18
+        // let window_id = SurfaceUserData::from_wl(surface).window_id;
+        // WaylandConnection::with_window_inner(window_id, |inner| {
+        //     if let Some(frame) = inner.window_frame.as_mut() {
+        //         frame.set_scaling_factor(new_factor as f64);
+        //     }
+        //     Ok(())
+        // });
```

Comment:
We're on SCTK 0.19 now, so should we go ahead and do this?

```suggestion
        let window_id = SurfaceUserData::from_wl(surface).window_id;
        WaylandConnection::with_window_inner(window_id, |inner| {
            if let Some(frame) = inner.window_frame.as_mut() {
                frame.set_scaling_factor(new_factor as f64);
            }
            Ok(())
        });
```

Reply:
yep, done